script.js

```diff
@@ -0,0 +1,480 @@
+          keyButton.addEventListener('click', () => {
+            this.properties.value += ' ';
+            this.triggerEvent();
+            this.myFocusFunction();
+          });
+          break;
+
+        case 'ArrowUp':
+          keyButton.classList.add('keyboard_key');
+          keyButton.textContent = '▲';
+          keyButton.addEventListener('click', () => {
+            this.properties.value += '▲';
+            this.triggerEvent();
+            this.myFocusFunction();
+          });
+          break;
+        case 'ArrowLeft':
+          keyButton.textContent = '◄';
+          keyButton.addEventListener('click', () => {
+            this.properties.value += '◄';
+            this.triggerEvent();
+            this.myFocusFunction();
+          });
+
+          break;
+        case 'ArrowDown':
+          keyButton.textContent = '▼';
+          keyButton.addEventListener('click', () => {
+            this.properties.value += '▼';
+            this.triggerEvent();
+            this.myFocusFunction();
+          });
+          break;
+        case 'ArrowRight':
+          keyButton.textContent = '►';
+          keyButton.addEventListener('click', () => {
+            this.properties.value += '►';
+            this.triggerEvent();
+            this.myFocusFunction();
+          });
+          break;
+
+        default:
+          if (this.properties.capsLock) {
+            keyButton.textContent = key.toUpperCase();
+          } else {
+            keyButton.textContent = key.toLowerCase();
+          }
+          keyButton.addEventListener('click', () => {
+            this.properties.value += this.properties.capsLock ? key.toUpperCase()
+              : key.toLowerCase();
+            this.triggerEvent();
+            this.myFocusFunction();
+          });
+
+
+          break;
+      }
+      fragment.appendChild(keyButton);
+
+      if (key === 'Backspace' || key === 'Enter' || key === '\\' || key === 'ShiftRight') {
+        fragment.appendChild(document.createElement('br'));
+      }
+    });
+    return fragment;
+  },
+  toggleCapsLock() {
+    const arr = document.querySelectorAll('.keyboard_key');
+    this.properties.capsLock = !this.properties.capsLock;
+    sessionStorage.setItem('capsLock', this.properties.capsLock);
+    for (let i = 0; i < arr.length; i += 1) {
+      if (/^[a-zA-Zа-яА-я]$/.test(arr[i].textContent)) {
+        arr[i].textContent = this.properties.capsLock ? arr[i].textContent.toUpperCase()
+          : arr[i].textContent.toLowerCase();
+      }
+    }
+  },
+  changeLanguage() {
+    setTimeout(() => {
+      this.deleteChild();
+      this.properties.eng = !this.properties.eng;
+      sessionStorage.setItem('eng', this.properties.eng);
+      this.createKeysAndListners();
+    }, 100);
+  },
+
+  deleteChild() {
+    const c = this.elements.keysContainer;
+    while (c.firstElementChild) {
+      c.firstElementChild.innerHTML = '';
+      c.removeChild(c.firstChild);
```

Comment:
It's not a good idea to delete each key any time you change the laguage. You just need to change its visible content as keys are always the same.